cmd/cmd.go

```diff
@@ -326,9 +327,36 @@ func StartNode(workingDir string, passwordFetcher func(*wallet.Wallet) (string,
 		crypto.XPrivateKeyHRP = "txsecret"
 	}
 
-	conf, err := config.LoadFromFile(PactusConfigPath(workingDir))
+	confPath := PactusConfigPath(workingDir)
+	conf, err := config.LoadFromFile(confPath, true)
 	if err != nil {
-		return nil, nil, err
+		PrintWarnMsg("Unable to load the config: %s", err)
+		PrintInfoMsg("Attempting to restore the config to the default values...")
+
+		// Let's backup the config first
+		confBack, err := config.LoadFromFile(confPath, false)
+		if err != nil {
+			return nil, nil, err
+		}
+
+		confBackupPath := fmt.Sprintf("%v_bak_%s", confPath, time.Now().Format("2006_01_02"))
+		err = os.Rename(confPath, confBackupPath)
+		if err != nil {
+			return nil, nil, err
+		}
+
+		switch gen.ChainType() {
```

Comment:
Add comment here, Now, we try to restore the config file with the nbumer of validators from old config